File: 25/25.py
def checkAbundant(n):
    sum=0
    for i in range(1,n):
        if n%i==0:
            sum+=i
    return sum

n=int(input("Enter a number:"))
num=checkAbundant(n)
if num>n:
    print("The given number is Abundant")
else:
    print("The given number is not Abundant")

File: 25/25.c
#include<stdio.h>

int checkAbundant(int n){
    int sum=0;
    for(int i=1;i<n;i++){
        if(n%i==0){
            sum+=i;
        }
    }
    return sum;
}
int main(){
    int n;
    printf("Enter a number:");
    scanf("%d",&n);
    int num = checkAbundant(n);
    if(num>n){
        printf("The given number is Abundant\n");
    }
    else{
        printf("The given number is not Abundant\n");
    }

}